templates/vdc/vdc.py

```diff
@@ -170,6 +178,64 @@ def disable(self):
         space.disable('The space should be disabled.')
         self.data['disabled'] = True
 
+    def portforward_create(
+                            self, 
```

Comment:
I don't like this kind of indentation, make it hard to read and ugly to look at.
I don't think the line is big enough to start splitting like this